x/concentrated-liquidity/position_test.go

```diff
@@ -2848,4 +2848,21 @@ func (s *KeeperTestSuite) TestNegativeTickRange() {
 			TotalIncentives:                       sdk.NewCoins(sdk.NewCoin("uosmo", expectedTotalIncentiveRewards.Ceil().TruncateInt())),
 		})
 	})
+
+	// Export and import genesis to make sure that negative accumulation does not lead to unexpected
+	// panics in serialization and deserialization.
+	spreadRewardAccumulator, err := s.clk.GetSpreadRewardAccumulator(s.Ctx, poolId)
+	s.Require().NoError(err)
+
+	accum, err := spreadRewardAccumulator.GetPosition(types.KeySpreadRewardPositionAccumulator(negativeIntervalAccumPositionId))
+	s.Require().NoError(err)
+
+	// Validate that at least one accumulator is negative for the test to be valid.
+	s.Require().True(accum.AccumValuePerShare.IsAnyNegative())
+
+	export := s.clk.ExportGenesis(s.Ctx)
+
+	s.SetupTest()
```

Comment:
why are we calling SetupTest here again?

Reply:
It clears any state created earlier so when we init gen we are doing so from scratch